tests/test_decorations.py

```diff
@@ -0,0 +1,111 @@
+from conftest import RawImage
+import pytest
+
+font_pool = [
+    '-*-fixed-medium-r-*-*-13-*-*-*-*-*-*-*',
+    'Dejavu Sans:pixelsize=12:bold'
+]
+
+
+def test_window_border_plain(hlwm, x11):
+    color = (0x9f, 0xbc, 0x12)
```

Comment:
Some colors are given in hex, some in decimals. Should we unify the format in this test suite?
And: what are those colors, do they come from the defaults or default autostart?

Reply:
The colors were chosen such that all integers within a test case are unique and not 0. Sometimes the hex is more convenient, sometimes decimals are. Hex-values correspond to the string representation of colors (e.g. `#9fbc12` here), but decimals are printed by pytest if an assertion fails. So I've used hex in test cases that worked out of the box and I've used decimals in test cases that needed much effort until they went through :-)

Reply:
Also, sometimes I have the impression that the colors are altered slightly, but I'm not sure (that's what I meant with the colorspace comment)

Reply:
I see, thanks for explaining!